### DISCUSSION TITLE
求助大佬我机子是5070ti，显存12g，为啥老显示爆显存该修改什么，问gpt说让我改启动参数但是不知道入口

### DISCUSSION THREAD

**Discussion by chenxizhu332** [hanamizuki-ai/launcher-support#135] [问答]
<img width="1970" height="1024" alt="屏幕截图 2026-02-25 153003" src="https://github.com/user-attachments/assets/f2ca8d44-4286-43fd-9bbd-d8a118d19ab3" />
<img width="1577" height="145" alt="image" src="https://github.com/user-attachments/assets/fb735ea7-453c-4ed9-b785-5da85234639e" />

**Comment by pratikrath126** [comment]
Hey! With a 12GB 5070ti, VRAM OOM usually comes from default settings being too large. Here are the most common fixes:

For Stable Diffusion (Automatic1111 or ComfyUI):
- Set --xformers flag to reduce memory usage
- Use --medvram or --lowvram if still OOM
- Change the TensorRT or attention implementation to xformers or flash_attention
- Reduce --max-batch-count and --max-batch-idx
- Use torch.cuda.empty_cache() after generations
- Reduce --width/--height or enable --freeze in ComfyUI

For training (LoRA/dreambooth):
- Decrease batch size to 1
- Use gradient checkpointing (--gradient-checkpointing)
- Use --precision full-fp16 or --no-half (sometimes half precision helps, sometimes hurts)
- Lower --train_batch_size and --micro_batch_size

Check your launcher's config file for VRAM_LIMIT or MAX_VRAM settings and lower it to something like 10G to leave headroom for system. Also ensure your CUDA/cuDNN versions match the application requirements.
